src/main/resources/org/jenkinsci/plugins/workflow/multibranch/Messages_fr.properties

```diff
@@ -0,0 +1,3 @@
+    ReadTrustedStep._has_been_modified_in_an_untrusted_revis=\u2018{0}\u2019 a été modifié dans une révision non fiable
+WorkflowMultiBranchProject.DisplayName=Pipeline Multibranches
+WorkflowMultiBranchProject.Description=Cr00e9e un ensemble de projets Pipeline en se basant sur les branches d00e9tect00e9es dans le d00e9p00f4t d'un gestionnaire de code source.
```

Comment:
Some of the digit sequences here look like they were intended to be escape sequences or something, for example, is `00e9` supposed to be `é`?

Reply:
There are indeed some problems here.
I will fix them and update the PR.